drivers/hmis/app/graphql/types/hmis_schema/ce_referral.rb

```diff
@@ -8,12 +8,19 @@
 
 module Types
   class HmisSchema::CeReferral < Types::BaseObject
+    # object is a Hmis::Ce::Referral
     field :id, ID, null: false
     field :opportunity, HmisSchema::CeOpportunity, null: false
     field :steps, [HmisSchema::CeReferralStep], null: false
     field :status, HmisSchema::Enums::CeReferralStatus, null: false
     field :client_id, ID, null: false
     field :client, Types::HmisSchema::Client, null: true
+    field :date_started, GraphQL::Types::ISO8601Date, null: false, method: :created_at
+    field :current_step, Types::HmisSchema::CeReferralStep, null: true
```

Comment:
If this is only for displaying in the table, perhaps `field :current_step_name, String` would be more appropriate? 

Especially given the comment below (_There can be multiple steps currently in progress, but we're only going to show one in the project referrals table_), since the logic for determining the current step has a heuristic like this, it seems like we _don't_ want to use the `currentStep` field for other purposes (say, for example, rendering the "current" step form on the referral page).